pkg/packagekit/updates.jsx

```diff
@@ -1182,41 +1183,41 @@ class OsUpdates extends React.Component {
     }
 
     loadUpdateDetails(pkg_ids) {
-        const limit = 500; // Load iteratively to avoid exceeding cockpit-ws frame size
-        PK.cancellableTransaction("GetUpdateDetail", [pkg_ids.slice(0, limit)], null, {
-            UpdateDetail: (packageId, updates, obsoletes, vendor_urls, bug_urls, cve_urls, restart,
-                update_text, changelog /* state, issued, updated */) => {
-                const u = this.state.updates[packageId];
-                if (!u) {
-                    console.warn("Mismatching update:", packageId);
-                    return;
-                }
+        const update_details = Object.assign(this.state.updates);
+
+        // load details for each package individually because of https://issues.redhat.com/browse/RHEL-109779
+        // otherwise we won't get details for any package if just a single one fails, and that pretty much always
+        // happens on RHEL
+        const promises = pkg_ids.map(pkg_id =>
+            PK.cancellableTransaction("GetUpdateDetail", [[pkg_id]], null, {
+                UpdateDetail: (packageId, updates, obsoletes, vendor_urls, bug_urls, cve_urls, restart,
+                    update_text, changelog /* state, issued, updated */) => {
+                    const description = this.removeHeading(update_text) || changelog;
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-22377-77217c12-20250829-142815-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)